lang/cs/Org.Apache.REEF.IO/PartitionedData/IPartition.cs

```diff
@@ -26,7 +26,7 @@ namespace Org.Apache.REEF.IO.PartitionedData
     /// </summary>
     /// <typeparam name="T">Generic Type representing data pointer.
     /// For example, for data in local file it can be file pointer </typeparam>
-    public interface IPartition<T>
+    public interface IPartition<T> : IDisposable
```

Comment:
Why is `IPartition` `IDisposable`? `IDisposable` should usually be a property of the implementation.